.github/workflows/scripts/generate_release_notes.rb

```diff
@@ -49,12 +59,21 @@ def build_release_content
       bugs: #{metadata[:bugs]}
       security: #{metadata[:security]}
       #{DIVIDER}
-
+      #{MAJOR_VERSION_BANNER % NewRelic::VERSION::MAJOR if major_bump?}
```

Comment:
Waiting to interpolate the constant until the metadata is built allows the test to use the stubbed constant instead of the real value.